### DISCUSSION TITLE
NextJS support

### DISCUSSION THREAD

**Discussion by gurustiven** [webstudio-is/webstudio-community#35] [Wishlist]
Hi! Congrats for the launch of this amazing product i can imagine unlimited possibilities for our startup with this integrated.

My question is a bout how to deploy this into a turborepo (nextjs) project i cannot find documentation about it, we are very interested on try it as self-hosted solution.

Again, congrats 👏

**Comment by kof** [comment]
Hi  Stiven,
Beta launch was all about a tool for visual developers and designers. We are basically building the foundation.
Integrations with custom code bases and sdk api for you to replace components and/or framework will follow, but isn't ready yet. You can try to replace  sdk-components-react-remix yourself and make it work, but its not gonna be trivial.
